Week3/homework/step2-3.js

```diff
@@ -4,9 +4,11 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  if (num > 0) {
```

Comment:
There is no need for this test as the loop body will not be executed if `num <= 0`.